margin.feature

```diff
@@ -405,48 +405,48 @@ Feature: Margin Protocol
       | Alice | 0  | $2    |
     Then margin balances are
       | Name  | Free  | Margin |
-      | Alice | $5000 | $4650  |
-    Then margin liquidity is $10350
+      | Alice | $5000 | $4649  |
+    Then margin liquidity is $10351
     When open positions
       | Name  | Pair   | Leverage | Amount | Price |
-      | Alice | EURUSD | Short 10  | $5000 | $2    |
+      | Alice | EURUSD | Short 10 | $5000  | $2    |
     Then margin balances are
       | Name  | Free  | Margin |
-      | Alice | $5000 | $4650  |
+      | Alice | $5000 | $4649  |
     And margin execute block 9..22
     When close positions
       | Name  | ID | Price |
       | Alice | 1  | $4    |
     Then margin balances are
       | Name  | Free  | Margin                 |
-      | Alice | $5000 | 4145485000000000000000 |
-    Then margin liquidity is 10854515000000000000000
+      | Alice | $5000 | 4751484900000000000000 |
```

Comment:
How can the balance increase? The price didn't change, so it must be 4649$ - marginSpreads - swapRates.

Reply:
This is Short 10x.
```
open_price = 3  - 0.02 = 2.98
close_price = 3  + 0.02 = 3.02
profit = leveraged_held * (close_price - open_price) 
-200 = 5000 * (3.02 - 2.98) 

accumulated_swap_usd_value = leveraged_debits * (accumulated_swap_rate - open_accumulated_swap_rate)) * price
302.4849 = 5000 * 2.98 * (0.030301 - 0.01)

so balance = balance - 200 + 302.4849 => 4649 + 102.48489999999998 = 4751.48489999999998 
```

Reply:
```
    And margin set swap rate
      | Pair    | Long | Short |
      | EURUSD  | -1%  | 1%    |
```

Reply:
Thats 200 loss, not profit. ;)

Reply:
If you do `4649-200-302.4849 = 4146.5151` is pretty close to the 4151 that I have. My guess would be the difference is in swap rate in USD vs swap rate in EUR? But I'd have to look deeper for that.

Reply:
i.e. if someone have a long/short EURUSD position, they get a -1% loss / 1% profit for each night holding such open position. e.g. $50 loss for a $5000 long position, and $50 profit for a $5000 short position.